vpd-tool/include/utils.hpp

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include "types.hpp"
+
+#include <nlohmann/json.hpp>
+#include <sdbusplus/asio/property.hpp>
+#include <sdbusplus/server.hpp>
+
+#include <iostream>
+
+namespace vpd
+{
+namespace utils
+{
+/**
+ * @brief An API to read property from Dbus.
+ *
+ * The caller of the API needs to validate the validatity and correctness of the
+ * type and value of data returned. The API will just fetch and retun the data
+ * without any data validation.
+ *
+ * Note: It will be caller's responsibility to check for empty value returned
+ * and generate appropriate error if required.
+ *
+ * @param[in] i_serviceName - Name of the Dbus service.
+ * @param[in] i_objectPath - Object path under the service.
+ * @param[in] i_interface - Interface under which property exist.
+ * @param[in] i_property - Property whose value is to be read.
+ * @return - Value read from Dbus on success.
+ *           empty variant for any failure.
+ */
+inline types::DbusVariantType readDbusProperty(const std::string& i_serviceName,
```

Comment:
this is duplicated API from dbus_utility.hpp 'readDbusProperty' .
To avoid any headers inclusion into vpd-tool, this API is being duplicated here.